COMMERCE-9442 - Unhandled error when saving an ERC that is already present

victorcdp opened the pull request:
Related Issue: [COMMERCE-9442](https://issues.liferay.com/browse/COMMERCE-9442)

Problem was that there was no frontend handling of the error in a lot of COMMERCE pages.
Proposed fix is to handle all the errors.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

katienesterovich commented:
ci:test:sf

katienesterovich commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-3.liferay.com/job/test-portal-source-format/2689/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-3.liferay.com//userContent/jobs/test-portal-source-format/builds/2689/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:ci:test:sf - 1 out of 1 jobs passed in 2 minutes 52 seconds 796 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/3d0ab205d7a0f58d9f6c893c34c909e550eec337">3d0ab205d7a0f58d9f6c893c34c909e550eec337</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/katienesterovich/liferay-portal/tree/LPS-103703">LPS-103703</a><br/>Branch GIT ID: <a href="https://github.com/katienesterovich/liferay-portal/commit/973119e0ff50b8775eba6e0cb03db52315a05dc0">973119e0ff50b8775eba6e0cb03db52315a05dc0</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-3.liferay.com/job/test-portal-source-format/2689/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-3.liferay.com/userContent/jobs/test-portal-source-format/builds/2689/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest(master)/4878/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-23.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4878/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:stable - 8 out of 9 jobs passed</h3><h3>:x: ci:test:relevant - 18 out of 20 jobs passed in 2 hours 23 seconds 95 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/aba855406499e5a51771ffa43b52d5c457efb6bd">aba855406499e5a51771ffa43b52d5c457efb6bd</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/a738cd2e983a714982ab81d8980a839e4ec8e625">a738cd2e983a714982ab81d8980a839e4ec8e625</a></p><details><summary><strong>ci:test:stable - 8 out of 9 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201283/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</strong></strike></a></li></ul></div><details><summary><strong>8 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/334424/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/334426/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201282/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/353863/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291094/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201278/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201276/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/334423/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 18 out of 20 jobs PASSED</strong></summary><div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest(master)/4878/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201283/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</strong></strike></a></li></ul></div><details><summary><strong>18 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201281/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/353860/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/334424/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/334426/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201282/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/353863/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291094/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201278/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/353859/">test-portal-acceptance-pullrequest-batch(master)/js-unit-private-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/353861/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201277/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/334425/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201280/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201276/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201279/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291093/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/353862/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/334423/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li></ul></details></details><h5>For more details click <a href="https://test-1-23.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4878/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201283/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></h5><div><h6>Job Results:</h6><p>3 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/201283//consoleText">AXIS_VARIABLE=0,label_exp=!master #201283</a><pre><code>     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:portal:portal-jmx-test:startTestableTomcat'.
     [exec] &gt; Timeout while waiting for the application server
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':apps:portal:portal-jmx-test:startTestableTomcat'.
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [exec] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [exec] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
     [exec] 	at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
     [exec] 	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
     [exec] 	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
     [exec] 	at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:34)
     [exec] 	at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.run(EventFiringTaskExecuter.java:51)</code></pre></div></li></ol></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html>

brianikim commented:
https://github.com/SamZiemer/liferay-portal/pull/683

CharlotteFrancis committed:
LRCI-3258 db2 batch versions

Luizcarlosqueiroz committed:
LPS-163069 Add json search macro

Luizcarlosqueiroz committed:
LPS-163069 Automate Object#CanSearchObjectEntriesViaAPI

izaera committed:
LPS-174464 Defer computation of legacy bridges

Instead of doing it in the tracker, do it all at once in _refreshJSModuleCaches().
Next step will be internalizing the computation of _processLegacyBridges() inside
JSModulesCache.

izaera committed:
LPS-174464 Do as promised in previous commit

Internalize _globalAliases inside JSModulesCache.

izaera committed:
LPS-174464 Internalize partialMatchMap into JSModulesCache

With this commit, all information regarding modules resolution is inside the
JSModulesCache.

Now we will extract JSModulesCache to its own class and decouple it from
NPMRegistry so that we can resolve modules without needing to access the
NPMRegistry several times.

This will prevent synchronization issues in the middle of module resolutions if
the NPMRegistry is updated.

shuyangzhou committed:
LPS-174464 Remove NPMRegistryUpdatesListener support, it is not used anymore.

(cherry picked from commit 5aabd517502e37155f8c649eb7f477fc2ca88dd3)

shuyangzhou committed:
LPS-175383 Hash the full NPMRegistryStateSnapshotImpl state

Since hashing everything vs hashing a subset + DXP fixes takes a similar amount
of time, go with full hash to be safer.

Before this commit, computing the digest took ~70ms, after the commit it takes
~90ms. It's in the same order of magnitude and small enough to prefer being safe
than fast.

izaera committed:
LPS-174464 Extract JSModulesCache

izaera committed:
LPS-174464 Move hash computation to JSModulesCache

izaera committed:
LPS-174464 Fix hash algorithm

izaera committed:
LPS-174464 Update hash algorithm

izaera committed:
LPS-174464 Move resolveJSPackageDependency to JSModulesCache

So that we don't need to expose the _cachedDependencyJSPackages.

izaera committed:
LPS-174464 Move mapModuleName to JSModulesCache

With this last change, we can make resolutions from JSModulesCache alone,
which will lets us return the proper resolutions for a given hash when
requested from "/o/js_resolve_modules/<hash>" without breaking the "cache
forever" semantics.

See https://github.com/brianchandotcom/liferay-portal/pull/129875#issuecomment-1423766299

izaera committed:
LPS-174464 Add a getter to obtain a snapshot of the NPMRegistry

This will allow us to obtain stable information for a given hash no matter
how many times we invoke the NPMRegistry and even if it gets updated in
between them.

Note that, if the NPMRegistry is updated, the returned resolution may fail
in the browser (because it will try to access back JS modules from a state
of the NPMRegistry that doesn't exist any more) but the contents of the CDN
or proxy caches will be correct so that, in the future, in case the portal
goes back to the state when the resolution was computed, the CDN/proxy
caches are not posisoned by an incorrect resolution.

Going back to a previous state shouldn't be unusual: think of the case when
something is deployed, then undeployed after testing it.

izaera committed:
LPS-174464 Use NPMRegistryStateSnapshot instead of live NPMRegistry

Make resolutions stable by using a snapshot of the registry, instead of
the live state, that may change several times during the resolution
process if something is deployed/undeployed or some fragment is modified.

izaera committed:
LPS-174464 Use NPMRegistryStateSnapshot from JSResolveModulesServlet

izaera committed:
LPS-174464 Rename JSModulesCache to NPMRegistryStateSnapshotImpl

izaera committed:
LPS-174464 Don't let NPMRegistryUpdates change/corrupt the active snapshot

Instead, save the calls so that NPMRegistry can then patch the snapshot in
_refreshJSModuleCaches().

izaera committed:
LPS-174464 Rename _refreshJSModuleCaches to _refreshNPMRegistryStateSnapshot

izaera committed:
LPS-174464 Fix redirect coming from server

If we don't add the query string, the AMD loader will not requery the
original resolution and will fail.

izaera committed:
LPS-174464 Fix NPMRegistryImpl service tracker filter

izaera committed:
LPS-174464 Fix param order

izaera committed:
LPS-174464 SF

Commit:
LPS-169905 SF, replace method new PersonalMenuEntryOrderComparator() with Collections.reverseOrder(new PropertyServiceReferenceComparator<>())

Commit:
LPS-169905 SF, inline

Commit:
LPS-169905 SF, simplify

Commit:
LPS-169905 SF, inline

Commit:
LPS-169905 SF, remove unnecessary logic in class PersonalMenuEntryRegistry(the property "product.navigation.personal.menu.group" will never be null because we use it as a filter)

Commit:
LPS-169905 SF, inline

MrJohn1911 committed:
LPS-177024 Change PlacedOrderItemDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177024 Change PlacedOrderDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177024 Change PlacedOrderAddressDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177024 Change PlacedOrderCommentDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177024 Change PlacedOrderItemShipmentDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177030 Change ChannelDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177030 Change WishListItemDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177030 Change PinDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177030 Change ProductSpecificationDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177030 Change ProductConfigurationDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177030 Change ProductOptionDTOConverter references to be DTOConverter

MrJohn1911 committed:
LPS-177030 Change ProductDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177030 Change MappedProductDTOConverter references to be DTOConverter

MrJohn1911 committed:
LPS-177030 Change RelatedProductDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177030 Change AttachmentDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177030 Change WishListDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177030 Change SkuDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177030 Change CategoryDTOConverter reference to be DTOConverter

rafaprax committed:
LPS-177044 Separate the ModelListener logic from the CommerceQualifier implementation. With this change, CommerceQualifier implementations do not need to be a ModelListener.

rafaprax committed:
LPS-177044 For each CommerceQualifierMetadata service register a modelListener service. This kept the old behavior and make ensures that for each CommerceQualifierMetadata instance there will be one modelListener to it.

rafaprax committed:
LPS-177044 Not needed anymore

daiazius committed:
LPS-177193 Change OptionValueDTOConverter references to be DTOConverter

daiazius committed:
LPS-177193 Change ProductOptionValueDTOConverter references to be DTOConverter

daiazius committed:
LPS-177193 Change CatalogDTOConverter references to be DTOConverter

daiazius committed:
LPS-177193 Change DiagramDTOConverter references to be DTOConverter

daiazius committed:
LPS-177193 Change PinDTOConverter references to be DTOConverter

daiazius committed:
LPS-177193 Change SpecificationDTOConverter references to be DTOConverter

daiazius committed:
LPS-177193 Change ProductSpecificationDTOConverter references to be DTOConverter

daiazius committed:
LPS-177193 Change ProductShippingConfigurationDTOConverter references to be DTOConverter

daiazius committed:
LPS-177193 Change ProductSubscriptionConfigurationDTOConverter references to be DTOConverter

daiazius committed:
LPS-177193 Change ProductConfigurationDTOConverter references to be DTOConverter

daiazius committed:
LPS-177193 Change ProductTaxConfigurationDTOConverter references to be DTOConverter

daiazius committed:
LPS-177207 Change OptionDTOConverter references to be DTOConverter

daiazius committed:
LPS-177207 Change ProductOptionDTOConverter references to be DTOConverter

daiazius committed:
LPS-177207 Change ProductGroupDTOConverter references to be DTOConverter

daiazius committed:
LPS-177207 Change ProductDTOConverter references to be DTOConverter

daiazius committed:
LPS-177207 Change MappedProductDTOConverter references to be DTOConverter

daiazius committed:
LPS-177207 Change GroupedProductDTOConverter references to be DTOConverter

daiazius committed:
LPS-177207 Change RelatedProductDTOConverter references to be DTOConverter

daiazius committed:
LPS-177207 Change ProductGroupProductDTOConverter references to be DTOConverter

daiazius committed:
LPS-177207 Change AttachmentDTOConverter references to be DTOConverter

daiazius committed:
LPS-177207 Change SkuDTOConverter references to be DTOConverter

daiazius committed:
LPS-177207 Change CategoryDTOConverter references to be DTOConverter

daiazius committed:
LPS-177207 Change OptionCategoryDTOConverter references to be DTOConverter

joshuacords committed:
LPS-176493 portal-search: FacetedSearcherImpl doesn't need to check and aligns with BaseIndexer

https://issues.liferay.com/browse/LPS-176493

joshuacords committed:
LPS-176493 portal-search-web: handle empty facet

https://issues.liferay.com/browse/LPS-176493

joshuacords committed:
LPS-176493 portal-search: FacetedSearcherImpl is no longer responsible for stopping empty searches

https://issues.liferay.com/browse/LPS-176493

CharlotteFrancis committed:
LRCI-3258 add extra wait condition before db2 startup

torresvictor100 committed:
LPS-173030 set _addLayoutPageTemplates

Commit:
LPS-169905 Apply(class PersonalMenuEntryRegistryImpl only sorts class PersonalMenuEntry when it collects class PersonalMenuEntry, class AccountRoleRequestHelper has no special requirements for the order of class PersonalMenuEntry, so in order to reduce the number of trackers for class PersonalMenuEntry, use class PersonalMenuEntryRegistry in class AccountRoleRequestHelper and RolesAdminPortlet to replace the original tracker)

joshuacords committed:
LPS-176493 portal-search: move check to SearcherImpl

https://issues.liferay.com/browse/LPS-176493

torresvictor100 committed:
LPS-173030 testIntegration

IstvanD committed:
LPS-177307 Adding the DDMField and DDMFieldAttribute table's missing indexes

IstvanD committed:
LPS-177307 Restructuring the queries to make use of indexing

IstvanD committed:
LPS-177307 Registering

MrJohn1911 committed:
LPS-177219 Change OrderRuleDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change OrderTypeDTOConverter referene to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change OrderRuleOrderTypeDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change TermOrderTypeDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change OrderNoteDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change ChannelDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change OrderRuleChannelDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change OrderTypeChannelDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change TermDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change AccountGroupDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change OrderRuleAccountGroupDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change OrderDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change BillingAddressDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change ShippingAddressDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change AccountDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change OrderRuleAccountDTOConverter reference to be DTOConverter

MrJohn1911 committed:
LPS-177219 Change OrderItemDTOConverter references to be DTOConverter

daiazius committed:
LPS-177193 Create DTOConverterConstants file

Commit:
LPS-178135 Remove unused interface SearchResultImageContributor

Commit:
LPS-178135 Apply

Commit:
LPS-178135 Remove unused variable searchResultImage

Commit:
LPS-178135 Remove unused interface SearchResultImage

Commit:
LPS-178135 Apply

Commit:
LPS-178135 Semantic versioning

torresvictor100 committed:
LPS-173030 change _addObjectDefinitions

joshuacords committed:
LPS-176493 portal-search: refactor for testing

https://issues.liferay.com/browse/LPS-176493

joshuacords committed:
LPS-176493 portal-search: add test

https://issues.liferay.com/browse/LPS-176493

lfbesada committed:
LPS-178316 Use first the ddmStructureId keeping as fallback ddmStructureKey

joshuacords committed:
LPS-176493 asset-test: fix AssetTagIndexerIndexedFieldsTest

https://issues.liferay.com/browse/LPS-176493

MrJohn1911 committed:
LPS-177030 Create constants file in headless-commerce-delivery-catalog-impl module

joshuacords committed:
LPS-176493 journal-test: fix JournalArticle tests

https://issues.liferay.com/browse/LPS-176493

joshuacords committed:
LPS-176493 users-admin-test: fix tests

https://issues.liferay.com/browse/LPS-176493

joshuacords committed:
LPS-176493 search-experiences-test: fix test

https://issues.liferay.com/browse/LPS-176493

Luizcarlosqueiroz committed:
LPS-163069 Add return statement when adding an entry

lfbesada committed:
LPS-178316 Adapt JournalArticle add and edit to use ddmStructureId instead of ddmStructureKey

lfbesada committed:
LPS-178316 Adds new methods to create JournalArticles but receiving the ddmStructureId instead of the ddmStructureKey

lfbesada committed:
LPS-178316 BuildService

lfbesada committed:
LPS-178316 Adds new methods to update JournalArticles but without receiving any structure-related parameter, as the JournalArticle can never change its structure once have been created.

We only have to set the structure-related fields when creating a new version and set the same structure as previous versions.

lfbesada committed:
LPS-178316 Deprecates the old methods

joshuacords committed:
LPS-176493 user-groups-admin-web-test: fix test

https://issues.liferay.com/browse/LPS-176493

Commit:
LPS-177869 Manager only can edit MDF in draft status

michaelhashimoto committed:
LRCI-3499 Add ability to deploy workspace zips

lfbesada committed:
LPS-178316 Fix typo

lfbesada committed:
LPS-178316 Update staging usages

jkappler committed:
LPS-178448 We want the field sets of relationships to be named with the pattern "Relation Label (Parent Object Definition Label)"", so change accordingly

lfbesada committed:
LPS-178316 Allow using ancestor site's structures

lfbesada committed:
LPS-178316 Remove deprecated methods

lfbesada committed:
LPS-178316 Replace usages of deprecated methods

lfbesada committed:
LPS-178316 Replace usages of deprecated methods in test classes and scripts

lfbesada committed:
LPS-178316 BuildService

daiazius committed:
LPS-176900 (headless-commerce-admin-account-impl) - DTOConverter implementations must be registered as a DTOConverter service only

daiazius committed:
LPS-176900 (headless-commerce-admin-account-impl) - Use a DTOConverter reference with a target filter instead of the specific DTOConverter class

daiazius committed:
LPS-177016 (headless-commerce-admin-channel-impl) - DTOConverter implementations must be registered as a DTOConverter service only

daiazius committed:
LPS-177016 (headless-commerce-admin-channel-impl) - Use a DTOConverter reference with a target filter instead of the specific DTOConverter class

daiazius committed:
LPS-177034 (headless-commerce-admin-shipment-impl) - DTOConverter implementations must be registered as a DTOConverter service only

daiazius committed:
LPS-177034 (headless-commerce-admin-shipment-impl) - Use a DTOConverter reference with a target filter instead of the specific DTOConverter class

victorg1991 committed:
LPS-165340 Remove not used parameter

izaera committed:
LPS-174464 gradle baseline

Commit:
ISSITES-5852 add categories/tags from Liferay to create app profile

ealonso committed:
LPS-178316 Keep javadocs

ealonso committed:
LPS-178316 It is not necessary to check permissions and use fetch

ealonso committed:
LPS-178316 Simplify

ealonso committed:
LPS-178316 classTypeId is the structureId so it is not necessary to calculate the DDMStructure

markocikos committed:
LPS-178260 Use `LinkOrButton`. In it, we use button when there is no `href`, fixing the missing focus.

joshitagaki committed:
LRAC-13090 Test fix update script to get search header values

BryanEngler committed:
LPS-176493 portal-search-web: consistent with other facetDCBuilders

BryanEngler committed:
LPS-176493 portal-search: simplify

Commit:
LPS-169905 Extract the logic of tracking class PersonalMenuEntry from class RolesAdminPortlet to class PersonalMenuEntryRegistryImpl

Commit:
LPS-169905 Add dependencies

Commit:
LPS-169905 Semantic versioning

sandrodw3 committed:
LPS-178851 It will get an object with selected keys

antonio-ortega committed:
LPS-178820 In case there is no FriendlyURL for Journal Content's default language, look for FriendlyURL for Site's default language before returning ArticleFriendlyURLException

CharlotteFrancis committed:
LRCI-3258 SF

victorhcunha committed:
LPS-169240 Fix UnpublishAndDeleteUploadedDefinition

victorhcunha committed:
LPS-169240 Remove Test UnpublishAndDeleteUploadedDefinition from quarantine

lfbesada committed:
LPS-178963 SF cache classNameIds in variables

lfbesada committed:
LPS-178963 Don't need to be static methods

lfbesada committed:
LPS-178963 SF rename

lfbesada committed:
LPS-178963 Adapt usages

Commit:
LPS-177869 Fixed status changing(Approved) after edited as Liferay manager

Commit:
LPS-176493 portal-search: Remove reference unbind

Commit:
LPS-177869 Create function to identify Liferay Manager

Commit:
LPS-177869 Changed permission to VIEW Role

Commit:
LPS-177869 Refactored Goals code

Commit:
LPS-177869 Refactored MDFRequest and SubmitForm code

michaelhashimoto committed:
LRCI-3475 Create task & task run class hierarchies to represent the liferay objects

michaelhashimoto committed:
LRCI-3475 Create a relationship between Build objects & Task objects

michaelhashimoto committed:
LRCI-3475 Add DALO classes to handle interactions with Task & Task Run objects

lfbesada committed:
LPS-178316 BuildService

shuyangzhou committed:
LPS-178616 Allow multi resource bundles to overwrite the same language.

linolaoj committed:
LPS-179068 shearching user by screen name

Commit:
LPS-177869 SF

victorhcunha committed:
LPS-179018 Fix CanExportPicklist Test

victorhcunha committed:
LPS-179018 Fix assertDownloadedFileName Macro

itstali committed:
LPS-178677 Fix RaylifeContactInfo macro

itstali committed:
LPS-178677 Fix RaylifeRetrieveQuote testcase

Commit:
LPS-177233 - Add exception handling

itstali committed:
LPS-178677 Fix RaylifeContactInfo macro SF

thaynam committed:
LPS-177816 Set Total mdf Request in objects

CalumR23 committed:
POSHI-504 Add getHtmlNodeText into Liferay Selenium

thaynam committed:
LPS-177816 Set Total mdf Request to be dinamic in Form

CalumR23 committed:
POSHI-504 Update com.liferay.poshi.core to 1.0.104

thaynam committed:
LPS-177816 Fix update MDF Request Activity in Salesforce

thaynam committed:
LPS-177816 Create delete budget

CalumR23 committed:
POSHI-504 Allow for use within user end poshi tests

thaynam committed:
LPS-177816 Remove comentary

pyoo47 committed:
LRCI-3258 SF

patriciajeanperez committed:
LPS-169226 Simplify steps by using JSON

michaelhashimoto committed:
LRCI-3475 Add space to match formatting of other JSON Strings

michaelhashimoto committed:
LRCI-3499 SF

ryanschuhler committed:
ISSITES-6006 Fix build and routes

Commit:
LPS-178316 Update Poshi side

lfbesada committed:
LPS-178767 Use the ThemeDisplay request and response as fallback

victorg1991 committed:
LPS-179010 Set the script as sandbox too

ruben-pulido committed:
LPS-177546 Create headless-site-test module

ruben-pulido committed:
LPS-177546 build.gradle

ruben-pulido committed:
LPS-177546 Add testDir to rest-config.yaml

ruben-pulido committed:
LPS-177546 buildREST

ruben-pulido committed:
LPS-177546 Create SiteResourceTest

DiegoHu97 committed:
LPS-160806 Fit background for preview images in Info Panel

ruben-pulido committed:
LPS-177546 Reverse sites list so child sites are deleted before parent sites

antonio-ortega committed:
LPS-178820 Considering site's default language since it seems more appropiate in this case

victorg1991 committed:
LPS-179145 Prevent default so that onClick event is not triggered

brianchandotcom committed:
LPS-174464 Rename

brianchandotcom committed:
LPS-174464 Rename

Commit:
LPS-179171 Fix CanCreateActionNotificationUsingOldValueWithOnAfterUpdateTriggerOnAccountObject

brianchandotcom committed:
LPS-173030 SF

brianchandotcom committed:
LPS-178616 Rename

brianchandotcom committed:
LPS-178316 as used

brianchandotcom committed:
LPS-178316 SF

brianchandotcom committed:
LPS-169905 SF

brianchandotcom committed:
LPS-176493 SF

patriciajeanperez committed:
LPS-169226 Add step

patriciajeanperez committed:
LPS-169226 Make this work in CE and DXP

brianchandotcom committed:
LRCI-3499 Don't break flow of other targets

brianchandotcom committed:
POSHI-504 prep next

brianchandotcom committed:
LPS-177546 SF

michaelhashimoto committed:
LRCI-3513 Ensure the jenkins events plugin is always subscribed on startup

shuyangzhou committed:
LPS-179240 Force equinox to resolve bundle by the caller thread to avoid unnecessary resolver threads creation.

michaelhashimoto committed:
LRCI-3512 Add failure message to help debug concurrent error

patriciajeanperez committed:
LPS-169226 Remove from quarantine

patriciajeanperez committed:
LPS-171875 Stabilize setup

patriciajeanperez committed:
LPS-171875 Remove from quarantine

patriciajeanperez committed:
LPS-171875 No need to skip portal instance

Commit:
COMMERCE-9442 Duplicate ERC error handling for Order Types

thaiscabrals committed:
LPS-175012 - Fix FormFields#FIELD_REMOVE

Commit:
COMMERCE-9442 Duplicate ERC error handling for Orders

pyoo47 committed:
LRCI-3512 Wordsmith

pyoo47 committed:
LRCI-3512 Print stack trace

pyoo47 committed:
LRCI-3512 Add another failure message if an IOException is thrown while getting the queue items JSON.

gamerson committed:
ISSITES-5302 Fix correct static includes

brianchandotcom committed:
LRCI-3258 SF

brianchandotcom committed:
ISSITES-5302 Can this work?

brianchandotcom committed:
LRCI-3512 prep next

liferay-translation committed:
LPS-77699 Update Translations

Commit:
COMMERCE-9442 Duplicate ERC error handling for Terms and Conditions

Commit:
COMMERCE-9442 Duplicate ERC error handling for Warehouses

Commit:
COMMERCE-9442 Duplicate ERC error handling for Price Lists/Promotions

Commit:
COMMERCE-9442 Duplicate ERC error handling for Discounts

Commit:
COMMERCE-9442 Duplicate ERC error handling for Product Groups

Commit:
COMMERCE-9442 Duplicate ERC error handling for Catalogs

Commit:
COMMERCE-9442 Change CProduct duplicate exception so it follows the pattern

Commit:
COMMERCE-9442 Duplicate ERC error handling for Products

Commit:
COMMERCE-9442 Duplicate ERC error handling for Product Options

Commit:
COMMERCE-9442 Duplicate ERC error handling for Channels

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

victorcdp commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/bc5bececfe9ee317711f15bbff3fccc3ee5af8cb">bc5bececfe9ee317711f15bbff3fccc3ee5af8cb</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/COMMERCE-9442">COMMERCE-9442</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/8f389ccaccf59024e1e5698b8a1adfbaba831e69">8f389ccaccf59024e1e5698b8a1adfbaba831e69</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-source-format/4567/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/4567/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-27.liferay.com/job/test-portal-source-format/4567/">test-portal-source-format#4567</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/4567/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/72">brianikim#72</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2241201409">[master] ci:test:sf - victorcdp &gt; brianikim - PR#72 - 2023-03-22[12:15:07]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-27.liferay.com/job/publish-testray-report/17071/">publish-testray-report#17071</a><br/></div></div>
